src/v2_services/chat_service.py

```
@@ -411,9 +434,74 @@ async def generate_answer(
+    if session_id:
+        search_query = await contextualize_question(query, session_id, llm)
+
+    # 번역
+    translated_query = await translate_query(search_query)
+
+    # 벡터 검색
+    docs, law_ids = await retrieve_laws(translated_query, country_id, db)
+
+    # 검색 결과 없을 시 안내
+    if not docs:
+        yield f"data: 죄송합니다. 질문하신 내용과 관련된 정확한 법률 정보를 찾을 수 없습니다. (관련도 낮음)\n\n"
```

Ruff (0.15.4)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)